code/game/supplyshuttle.dm

```diff
@@ -1,667 +0,0 @@
-//Config stuff
```

Comment:
Is deleting this entire file intended?

Reply:
yeah its not needed anymore

Reply:
Could use some atomization but it doesn't hurt I suppose.